starters/next-routing-chakra-ui/src/pages/index.tsx

```diff
@@ -0,0 +1,46 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Link from 'next/link';
+import { Heading, Box, Center, UnorderedList, ListItem, Flex } from '@chakra-ui/react';
+
+type Routes = {
+  text: string;
+  route: string;
+};
+
+const Home: NextPage = () => {
+  const routesArr: Routes[] = [
+    { text: 'See Counter example component', route: '/counter-example' },
+    { text: 'See Fetch example component', route: 'fetch-example' },
```

Comment:
Question: for the first route you have a `/` before the name, and the second one you don't. I feel like these need to be consistent? Or are they nested differently?

Reply:
Yeah, those should be consistent. 
Good catch